### ISSUE TITLE
How to change center target in Helix SharpDX?

### ISSUE THREAD

**Issue by mk48** [helix-toolkit/helix-toolkit#310]
In helix toolkit double right click can change the center target. How can I do the same in Helix SharpDX?

I tried like the following but the Target is not changed.

```
<hx:Viewport3DX.InputBindings>
       <MouseBinding Gesture="MiddleClick" Command="hx:ViewportCommands.SetTarget"/>
</hx:Viewport3DX.InputBindings>
```

**Comment by Timmitry** [comment]
I can confirm this bug, and created a [pull request](https://github.com/helix-toolkit/helix-toolkit/pull/577) which should hopefully fix the issue.

**Comment by holance** [comment]
Fixed in #577
